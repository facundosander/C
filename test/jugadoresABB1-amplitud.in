crearArbolJugadores

# Construyo arbol 

# -- Primer jugador Juan 1 --
crearJugada
1 1 2 3 4
agregarEnPartida
crearJugador 1 21 Uruguay Juan
insertarJugadorArbolJugadores

esVacioArbolJugadores

# -- Segundo jugador Roberto 5 --
crearJugada
1 4 3 2 1
agregarEnPartida
crearJugador 5 19 Brasil Roberto
insertarJugadorArbolJugadores

# -- Tercer jugador Luis 7 --
crearJugada
1 1 1 1 1
agregarEnPartida
crearJugador 7 21 Uruguay Luis
insertarJugadorArbolJugadores


# -- Cuarto jugador Andres 8 --
crearJugada
1 3 3 3 3
agregarEnPartida
crearJugador 8 19 Uruguay Andres
insertarJugadorArbolJugadores

# -- Quinto jugador Emilia 2 --
crearJugada
1 1 1 1 1
agregarEnPartida
crearJugador 2 21 Uruguay Emmilia
insertarJugadorArbolJugadores


# -- Sexto jugador Ines 10 --
crearJugada
2 2 2 2 2
agregarEnPartida
crearJugador 10 21 Uruguay Ines
insertarJugadorArbolJugadores


# -- Septimo jugador Isabel 3 --
crearJugada
1 1 1 1 1
agregarEnPartida
crearJugador 3 26 Uruguay Isabel
insertarJugadorArbolJugadores

# La maxima amplitud es 2
amplitudArbolJugadores

liberarArbolJugadores

Fin