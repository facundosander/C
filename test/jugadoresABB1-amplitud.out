1>2>#  Construyo arbol .
3>#  -- Primer jugador Juan 1 --.
4>5>6>7>8>El árbol de jugadores NO está vacío.
9>#  -- Segundo jugador Roberto 5 --.
10>11>12>13>14>#  -- Tercer jugador Luis 7 --.
15>16>17>18>19>#  -- Cuarto jugador Andres 8 --.
20>21>22>23>24>#  -- Quinto jugador Emilia 2 --.
25>26>27>28>29>#  -- Sexto jugador Ines 10 --.
30>31>32>33>34>#  -- Septimo jugador Isabel 3 --.
35>36>37>38>39>#  La maxima amplitud es 2.
40>La amplitud del árbol es: 2.
41>42>Fin.
